_ext/gradle/java-setup.gradle

```diff
@@ -1,78 +1,19 @@
 //////////
 // JAVA //
 //////////
-repositories {
-	mavenCentral()
-	jcenter()
-	/* Allow access of local SNAPSHOT builds. Sanity of dependencies
-	   is guranteed by the lib-extra build. */
-	mavenLocal()
-}
+apply from: rootProject.file('gradle/java-setup.gradle')
 
 // setup java
 apply plugin: 'java'
 
-sourceCompatibility = ext_VER_JAVA
-targetCompatibility = ext_VER_JAVA
+// Show warning locations
 tasks.withType(JavaCompile) { 
-	options.encoding = 'UTF-8'
```

Comment:
Why not `UTF-8`?

Reply:
Clean-up
UFT-8 required e.g. for:
* author names (as class comments)
* code comments in foreign language
* multi language UI

Think I once needed it when I refereed to the origin of "stolen" Eclipse code.
Due to service support / stubbing of `eclipse-base` such ugly work-arounds should not be required anymore.

Reply:
We have `UTF-8` enabled for the main projects: https://github.com/diffplug/spotless/blob/e870e78999887595d8480c699c4ef71c9e974b32/gradle/java-setup.gradle#L11

I believe it is needed due to string constants used in the diff formatter, if memory serves.  It's okay for `_ext` to be different, I'll let you decide.